_locales/en/messages.json

```diff
@@ -0,0 +1,103 @@
+{
+  "Name": {
+    "message": "SponsorBlock",
+    "description": "Name of the extension."
+  },
+  "fullName": {
+    "message": "SponsorBlock for YouTube - Skip Sponsorships",
+    "description": "Name of the extension."
+  },
+
+  "Description": {
+    "message": "Skip over sponsorship on YouTube videos. Report sponsors on videos you watch to save the time of others.",
+    "description": "Description of the extension."
+  },
+  "helpPage": {
+    "message": "index_en.html"
+  },
+  "400": {
+    "message": "Server said this request was invalid"
+  },
+  "429": {
+    "message": "You have submitted too many sponsor times for this one video, are you sure there are this many?"
+  },
+  "409": {
+    "message": "This has already been submitted before"
+  },
+  "502": {
+    "message": "It seems the server is down. Contact the dev to inform them."
+  },
+  "ChannelWhitelisted": {
+    "message": "Channel Whitelisted!"
+  },
+  "Sponsor": {
+    "message": "Sponsor"
+  },
+  "Sponsors": {
+    "message": "Sponsors"
+  },
+   "Segment": {
+    "message": "sponsor segment"
+  },
+  "Segments": {
+    "message": "sponsor segments"
+  },
+  "Dismiss": {
+    "message": "Dismiss"
+  },
+  "Hide": {
+    "message": "Don't Show This Again"
+  },
+  "HitGoBack": {
+    "message": "Hit go back to get to where you came from."
+  },
+  "GoBack": {
+    "message": "Go back"
+  },
+  "ConfirmMSG": {
+    "message": "\n\nTo edit or delete individual values, click the info button or open the extension popup by clicking the extension icon in the top right corner."
+  },
+  "ClearThis": {
+    "message": "Are you sure you want to clear this?\n\n"
+  },
+  "Unknown": {
+    "message": "There was an error submitting your sponsor times, please try again later."
+  },
+  
+  "SponsorFound": {
+    "message": "This video's sponsors are in the database!"
+  },
+  "Sponsor404": {
+    "message": "No sponsors found"
+  },
+  "SponsorStart": {
+    "message": "Sponsorship Starts Now"
+  },
+  "SponsorEND": {
+    "message": "Sponsorship Ends Now"
+  },
+  "noVIDEOID": {
+    "message": "This probably isn't a YouTube tab, or you clicked too early. \n If you know this is a YouTube tab,\n close this popup and open it again."
+  },
+  "VOTED": {
+    "message": "Thanks for voting!"
+  },
+  "VOTE_FAIL": {
```

Comment:
Why is some stuff all caps, some stuff `PascalCase` and some stuff `camelCase`?

Reply:
Because theres no linter to stop me xD

Reply:
You are purposefully changing format randomly?

Reply:
no there was no rules in place for json files

Reply:
Nope its changed now